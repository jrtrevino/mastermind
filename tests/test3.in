43673473463
Z
8
50
AKIMIHLE 
LMAPBDND
IHJFFJAK
HHGEKFJE
GKGJMKDK
KEKKFELE
OKABBKFP
DIDLGAFI
CNLKLCFL
JLACBOIJ
DEKAIOIH
DDAOGAPP
DKKAEFBF
CAOMEDEI
GPLKKFAP
CJLHEMOK
KHBKNEAA
CCPPJLPM
FFNGECEI DNLFJEEH OFFMCLCI NJNJAMCI JKFEGNOK APFACFDJ EENMBAFP GAPEHFDI MKGHMAFN KOCIOABG NMAPGKPC HJNMKBBP JGHFHOKB BGLKOADJ PDHIDDEI IANOMAKG ALNMKNNM IAMMDJCM HEPFOANK BAEKJMHN LMEBFCBM FKLDAGJA FJCKDBFC DEABAEEG KJOHCCPH ADEBHMHM EDDDAJCH LBNKKMFH NAIFJCAB KJBBNKFO FBPNMADC
DBPMKJNH 
