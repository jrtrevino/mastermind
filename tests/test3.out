Enter the seed for random number generation: Enter the maximum letter for the game (A-Z): Enter the number of positions for the game (1-8): Enter the number of guesses allowed for the game: 
Enter guess 1: Nope, 0 exact guesses and 3 inexact guesses

Enter guess 2: Nope, 0 exact guesses and 2 inexact guesses

Enter guess 3: Nope, 0 exact guesses and 1 inexact guesses

Enter guess 4: Nope, 0 exact guesses and 3 inexact guesses

Enter guess 5: Nope, 0 exact guesses and 0 inexact guesses

Enter guess 6: Nope, 1 exact guesses and 2 inexact guesses

Enter guess 7: Nope, 0 exact guesses and 1 inexact guesses

Enter guess 8: Nope, 0 exact guesses and 1 inexact guesses

Enter guess 9: Nope, 0 exact guesses and 2 inexact guesses

Enter guess 10: Nope, 0 exact guesses and 2 inexact guesses

Enter guess 11: Nope, 1 exact guesses and 2 inexact guesses

Enter guess 12: Nope, 1 exact guesses and 0 inexact guesses

Enter guess 13: Nope, 0 exact guesses and 1 inexact guesses

Enter guess 14: Nope, 0 exact guesses and 3 inexact guesses

Enter guess 15: Nope, 0 exact guesses and 1 inexact guesses

Enter guess 16: Nope, 0 exact guesses and 4 inexact guesses

Enter guess 17: Nope, 0 exact guesses and 3 inexact guesses

Enter guess 18: Nope, 1 exact guesses and 0 inexact guesses

Enter guess 19: Nope, 0 exact guesses and 3 inexact guesses

Enter guess 20: Nope, 0 exact guesses and 2 inexact guesses

Enter guess 21: 